### DISCUSSION TITLE
Nemo - recent folder history like Caja

### DISCUSSION THREAD

**Discussion by HT-7** [linuxmint/discussions#432] [Ideas]
Caja file manager shows a list of recently opened directories.

That list can be viewwed in the "Go" menu, which is located below the title bar of the window. Also the ES File Explorer for Android has this since the early 2010s. This is different from the navigation history (right click on previous/next arrows in the top left), because it would allow quickly re-opening a location of any closed tab.

It would be nice to see this feature in Nemo.

Thank you for considering my suggestion.

**Comment by Jopp-gh** [comment]
I thought to find your feature as a core applet. But there isn't one, only an applet for recent files.

I subscribe your request. Maybe you may be interested to read my proposal of modern browsing ?https://github.com/orgs/linuxmint/discussions/430
